tests/attribute_annotations.pyi

```diff
@@ -1,16 +1,23 @@
-field1: int
-field2: int = ...
+field0: int
+field1: int = ...  # Y032 Default value unnecessary for module-level attribute
+field2: int = 0  # Y032 Default value unnecessary for module-level attribute
 field3 = ...  # type: int
 field4: int = 0  # Y015 Attribute must not have a default value other than "..."
 field5 = 0  # type: int  # Y015 Attribute must not have a default value other than "..."
 field6 = 0  # Y015 Attribute must not have a default value other than "..."
 field7 = b""  # Y015 Attribute must not have a default value other than "..."
+field8: str = ...
+
+thing_using_field3: field3
+thing_using_field4: field4
+thing_using_field5: field5
+thing_using_field6: field6
+thing_using_field7: field7
```

Comment:
These would emit errors in type checkers, because they use an object as a type. How about `thing_using_field3 = field3.to_bytes` or something?